### ISSUE TITLE
Default post visibility setting not being honored

### ISSUE THREAD

**Issue by jeffsikes** [elk-zone/elk#1926]
I have an account where I want all posts to be set to "Unlisted" by default. Mastodon offers a setting for this in the preferences section.

![image](https://user-images.githubusercontent.com/6627582/227801335-adcd6007-a0b7-4bef-9c71-9c5de5421093.png)

This setting is honored by other apps like Ice Cubes and Ivory. However, it doesn't seem to be utilized in Elk. Just to be sure, I logged in under private mode and logged in with only this account (it's a secondary account to my main). It still chose "Public" visibility by default.

<img width="714" alt="image" src="https://user-images.githubusercontent.com/6627582/227801904-fde6f7bd-2819-4dc8-b743-f3297f479e2b.png">

There is an API endpoint to pull the preference here, both in the official API and in Masto.js:

https://docs.joinmastodon.org/methods/preferences/
https://neet.github.io/masto.js/classes/mastodon.v1.PreferenceRepository.html

There's a similar issue posted regarding choosing a default for REPLIES, but this is for setting the default for ALL posts as defined in the account's preferences.

https://github.com/elk-zone/elk/issues/760

Thanks!

**Comment by rpavlik** [comment]
Can confirm: my default in the standard web app is unlisted, but Elk is stuck on Public as the default.  0.7.5 at least has the issue.

**Comment by skiant** [comment]
This seems like a duplicate of #1040 and it should be fixed by #1733 unless I'm missing something?

**Comment by boehs** [comment]
> This seems like a duplicate of #1040 and it should be fixed by #1733 unless I'm missing something?

#1040 was closed by #1699. If this issue is fixed when #1733 is merged, let's close this one

**Comment by patak-dev** [comment]
@jeffsikes or others in this issue, would you try https://main.elk.zone to see if this issue has been resolved?

**Comment by jeffsikes** [comment]
Confirmed as working. I have 3 accounts attached to Elk. Switching between them, I see the default posting visibility updated as expected on each one. 

Tried posting from each one, and verified that the posts had the appropriate visibility set. 

I tried Unlisted and Followers Only.

My only note is that if you CHANGE your default posting visibility on your instance's Preferences -> Other section, then come back into Elk and refresh, it doesn't update. 

I think that is expected, you don't want to be constantly checking for the visibility preferences. But if there is any help documentation about the feature, would be worth mentioning that any updates to your default posting visibility on your server's profile requires you to log out and back in to Elk for it to take affect.

A separate enhancement might be a "Refresh my Account settings" feature in the Settings -> Profile menu. "Refresh your account settings from [your server]."

Thanks for fixing this! 🎉
